src/user/models/user-data/user-data.service.ts

```diff
@@ -66,7 +70,8 @@ export class UserDataService {
     let userData = await this.userDataRepo.findOne(userDataId);
     if (!userData) throw new NotFoundException('User data not found');
 
-    userData = await this.updateUserSettings(userData, dto);
+    // Is this needed here?
+    userData = await this.updateUserSettings(userData, {});
```

Comment:
I think so 😉  what are your thoughts? And why `updateUserSettings(userData, {})`?